app/controllers/media_objects_controller.rb

```diff
@@ -417,8 +417,12 @@ def update_status
     Array(params[:id]).each do |id|
       media_object = MediaObject.find(id)
       if cannot? :update, media_object
-        errors += ["#{media_object.title} (#{id}) (permission denied)."]
+        errors += ["#{media_object&.title} (#{id}) (permission denied)."]
       else
+        unless media_object.title.present? && media_object.date_issued.present?
```

Comment:
What about moving this check into the handling for 'publish' so 'unpublish' can happen even if the item is invalid?  Although in order to allow that there probably need to be changes made to `publish!` for skipping validations when unpublishing.

Reply:
Well, with this manual check the active record validations are being rendered redundant for `publish!` anyways so it may be the move to disable validations on `publish!` in general? I'm not really happy about that, but it is the logical progression with this approach...